src/fe-text/gui-readline.c

```diff
@@ -306,7 +315,14 @@ static void paste_send(void)
 		}
 	}
 
-	gui_entry_set_text(active_entry, str->str);
+	if (paste_was_bracketed_mode) {
+		/* the text before the bracket end should be sent along with the rest */
```

Comment:
if you have paste something like this:

     \x1b[200~text1
    text2 text3 text4
    text5 text6 text7
    text8 text9\x1b[201~

you wouldn't want to answer the confirmation message, send the first three lines to the channel, and be left with `text8 text9` in the input box